### PR TITLE
docs: Clay CSS 3 Recipes Clay Range

### PR DESCRIPTION
**Pull request by pat270**

### CODE REVIEW

File: css/clay_css_2_recipes/06_clay_range.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+	thumb-bg: $primary,
+	thumb-box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2)#{','} 0px 2px 2px 0px rgba(0, 0, 0, 0.14)#{','} 0px 1px 5px 0px rgba(0, 0, 0, 0.12),
+	thumb-height: 1.25rem,
+	thumb-width: 1.25rem,
+	track-bg: $primary-l4,
+	progress-height: 0.375rem,
+);
+```
+
+> One thing to note is the use of [Sass interpolation](https://sass-lang.com/documentation/interpolation) for commas in `thumb-box-shadow`, this is required when passing multiple `box-shadow`s in a Sass map. Interpolating comma prevents Sass from confusing it with the end of a key value pair. This also applies to any CSS property that delimits multiple properties with comma (e.g., `background` and `transition`).
```
Comment (wincent) [inline css/clay_css_2_recipes/06_clay_range.md:20]: "Interpolating comma" looks like it should be one of:

- "Interpolating the comma"
- "Interpolating commas"

Similarly, it looks like "that delimits multiple properties with comma" should be "that delimits multilple properites with commas". 🤔?

File: css/clay_css_2_recipes/06_clay_range.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+## Clay Range Hover
+
+Material Design Continuous Sliders hover state displays a halo around the thumb. We will need to add custom styles in `/src/css/_clay_custom.scss` since the mixin `clay-range-input-variant` only has options to modify `hover-cursor` and `hover-thumb-bg`. We will use the `::before` pseudo element to apply our hover styles. We create a transparent button overlay with a box-shadow for the halo.
```
Comment (wincent) [inline css/clay_css_2_recipes/06_clay_range.md:24]: Should that be?

> Material Design Continuous Slider's hover state

**Review by wincent — APPROVED** [review]
Looks good to me with some tiny tweaks suggested inline.

**Comment by pat270** [comment]
Made the changes, thanks @wincent !

**Comment by marcoscv-work** [comment]
LGTM!!!

**Pull request merged**